<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dialogPlace</class>
 <widget class="QDialog" name="dialogPlace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>213</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Place</string>
  </property>
  <widget class="QPushButton" name="buttonOKPlace">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>170</y>
     <width>93</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>371</width>
     <height>131</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>91</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Place tokens</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
